src/redux/books/books.js

```diff
@@ -1,7 +1,10 @@
 const ADD_BOOK = 'bookstore/books/ADD_BOOK';
 const DELETE_BOOK = 'bookstore/books/DELETE_BOOK';
 
-const initialState = [];
+const initialState = [
+  { title: 'book one', author: 'ben' },
+  { title: 'book two', author: 'Nick' },
+];
```

Comment:
- [ ] (OPTIONAL) Great job so far 👍 Kindly provide a unique id to each of these books or simply remove them from the initial state as they are currently causing an error on the browser console 😄 
- Console error:
![Annotation 2022-05-03 193217](https://user-images.githubusercontent.com/85113567/166519682-420593d1-2819-4eb7-bda8-ca743c1ddd3b.jpg)

Reply:
Thanks for the review, I will delete those books today when I work with the API